### ISSUE TITLE
Quote request: Upload "Press Release" content

### ISSUE THREAD

**Issue by gatsby8870** [snehal-addweb/Mortgagespeak#96]
Hello! I would like a quote on the below task, please.

Thank you!

-----

Goal:
Enable select users to add "press release" content.

Details:
My core clients submit press releases that I am too busy to upload. I'd like them to upload their own press releases and have it show their "top of page" ad. These clients would create, edit and delete any content they create. No other new functionality is requested with this quote.

The page I use, Alpha Content, isn't very user-friendly to share and it has too many items on it.

Fields to Show on new page:
Title field
Pop Up Text (show existing 250 character counter)
HTML Body Summary

The other required fields can be hard-coded/invisible to them such as:
already checked box for press release
already tagged for their company tag ( I can assign these)
already selected for their top of page ad
already selected for LinkedIN Recommended "no"
Article Source already done ("MortgageSpeak")
Save = Publish
Edit and Delete functionality

**Comment by roshni-addweb** [comment]
Hi Greg,

Please confirm which Ad will be selected for Top page ad when I am adding ads to Custom Ads field in Alpha content or I select the 1st ad from the Top Page Ad list for every user who is adding press release content 

Thanks

**Comment by gatsby8870** [comment]
Hello.

Each client currently has their own top page ad now. I'd just want the right client's ad on their press release content.

For example:

Client XYZ shows "top page ad XYZ"
Client ABC shows "top page ad ABC".

Thank you!


Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by roshni-addweb** [comment]
Hi Greg,

Thanks for your quick response

Right now you are selecting which ad will be shown on which press-release content, there is nothing like particular Simple Ad is created for this particular user i.e we cannot identify the ad content for specific user.
Do you want me add a field in Simple Ads content type and where you can select user? So I can get that Ad and displayed it top of that user content (Press Releases)

Thanks

**Comment by gatsby8870** [comment]
Yes, that sounds exactly like what we would need. Please add that to the estimate.


Thank you and I hope you are doing great!


________________________________

**Comment by gatsby8870** [comment]
Hello.

Just looking for an estimate on this idea.

Thank you.

**Comment by roshni-addweb** [comment]
Hi Greg,

It would require 10-12 hrs approx. Please take a note that the new field to select user in Simple Ads content type will reflect to Press release content - Top Page only.

Thanks

**Comment by gatsby8870** [comment]
Ok, please begin.

I want to make sure the process flow is that a client of mine can log in, copy/paste their press release and their top banner ad would be displayed on that content page. As an admin, I would assign the top page ad to the user so it shows that top page ad.

I am looking for something easy where my clients can login, copy/paste, then share their content, edit or delete it as needed. The brand at the top of the page as well.

Thank you.

**Comment by roshni-addweb** [comment]
Hi Greg,

Please check this Implementation on live. We Created a New role called 'Press Release'
Can you please tell me that where you want to add the link for "Add Press Release"  for the "Press Release" Role user. Currently, you have to access it directly (http://mortgagespeak.com/node/add/alpha-content) by hitting it (Only for Press Release users).

User Reference field is created in Simple Ad content type where you have to select one user for that particular Ad.
**NOTE:** if a Simple Ad is not assigned to the particular user then he will not be able to see any custom ads on that press release content page.

1. already checked box for press release 
  >> Done.
2. already tagged for their company tag ( I can assign these)
  >> Done, If a user has his assigned company then while editing the content it will be shown selected. 
3. already selected for their top of page ad
  >> Done If a 'simple Content ad' is assigned to that user then & only then that ad will be shown selected.
4. already selected for LinkedIN Recommended "no"
  >> Done
5. Article Source already done ("MortgageSpeak")
  >> Done
6. Save = Publish
  >> Done
7. Edit and Delete functionality
  >> Done

Thanks

**Comment by gatsby8870** [comment]
Hello!
Thank you so much!

I moved apartments and I've been without the internet for a while.

Everything looks good but can you send me the link to test?

I've assigned an as needed but I cannot test the user experience without the url for content creation/submission.

Thank you!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by gatsby8870** [comment]
Thanks, I figured out the url: alpha content.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by gatsby8870** [comment]
I'll find a place for the link tomorrow!

Have a great day and thank you for your hard work.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by gatsby8870** [comment]
Please change the big black font (page header) on this page from "Create Alpha Content" to "Create Content". This is only a change in the page header, not url. It is b/c "Alpha Content" might confuse users who are coming here to upload their "press release".

That's it!

**Comment by gatsby8870** [comment]
![20180322_214239 1](https://user-images.githubusercontent.com/18587752/37809654-3ca41808-2e1e-11e8-82f5-90d399c7a72f.jpg)

Here is how I see the page name looking, to be helpful.

**Comment by roshni-addweb** [comment]
Hello Greg,

It is done. Please check and let me know where should I add link for creating Press Releases.

Thanks

**Comment by gatsby8870** [comment]
Hello. Everything is looking really good. My clients used it for the first time today and found it simple and effective. Thank you!

One thing I would like to add is the ability for the user to tag companies that are mentioned in the release.

Please add the "Company Tag" selection field so the user may select their own company as well as their clients featured in their press release and have that uploaded content shown in the respective "Company Newsrooms".

Thank you!

**Comment by roshni-addweb** [comment]
Hi Greg,

It is done over live site. Please check.
Further, all company tags which is assigned to user will be preselected and user will able to select/deselect the tags. 

Thanks

**Comment by gatsby8870** [comment]
Can you please increase the height of the "Company Tag" selection field?

Question: Is it required that company tags which is assigned to user will be preselected? As the user uploads different press release stories, each release will have different companies mentioned. So the preselection doesn't truly help them. Can we just have this done like as done for the admin (me)? There is no preselected company and they just select the applicable company tags for that selected piece of content.

Thank you.

**Comment by roshni-addweb** [comment]
Hi @gatsby8870,

Please check above comment on live.

Thanks

**Comment by gatsby8870** [comment]
Hi, looks good.

It seems, however, the Alpha Content page header we took down is back. See attached image.

Can you remove leaving only "Create Content"? My clients are coming to this page to upload a press release so "Alpha Content" will confuse them.

Thank you.
[Image]




Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by roshni-addweb** [comment]
Hi @gatsby8870,

Please check above comment on live.

Thanks
